catalog/app/components/JsonEditor/Cell.js

```diff
@@ -178,7 +179,7 @@ export default function Cell({
     <div
       className={cx(classes.root, { [classes.disabled]: !isEditable })}
       role="textbox"
-      tabIndex={0}
+      tabIndex={isEditable ? null : 0}
```

Comment:
you sure this should be `null` for editable fields and `0` for non-editable, not the other way around?

Reply:
:(